Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMLearnConfigWizardPage.xaml.cs

```diff
@@ -177,6 +243,14 @@ private void ClearAutoMapElementTypesSection()
             xAutoMapElementTypesGrid.DataSourceList = mWizard.AutoMapElementTypesList;
         }
 
+        private void ClearAutoMapElementLocatorsSection()
+        {
+            xAutoMapElementLocatorsExpander.IsExpanded = false;
+            xAutoMapElementLocatorsExpander.IsEnabled = false;
+            mWizard.AutoMapElementLocatorsList = new ObservableList<ElementLocator>();
```

Comment:
we don't really need to clear the xAutoMapElementTypesGrid.DataSourceList here, not sure why it exist for ClearAutoMapElementTypesSection()

Reply:
I think to toggle some Section related Didplay properties. Shall i modify that too ?